### PR TITLE
performance-impls: LCP should use startTime

### PR DESCRIPTION
**Pull request by samouri**
**summary**
There isn't much utility in separately tracking _LCP Load_ and _LCP Render_.

- `loadTime` corresponds to when the resource associated with a node has been loaded from network. It is always available on every node, but is relatively meaningless for any non-remote resource on a page. For example, all `<h1>` tags have a loadTime, but that time is 0. Therefore if a text node were chosen as the LCP Element, we'd report 0 for LCP-L. 
- `renderTime` corresponds to when the node is painted to the screen. This has a meaningful value for text nodes, but ends up being absent for any crossorigin resource without a `Timing-Allow-Origin` header. In these cases it is recommended to fallback to loadTime since the numbers will be relatively close to each other.  Since AMP Caches usually host all their images, this is also not a very strong factor for AMP.

cc @ampproject/wg-performance

### CODE REVIEW

**Comment by erwinmombay** [comment]
Fixes https://github.com/ampproject/amphtml/issues/34383

File: test/unit/test-performance.js [hunk 1]
```diff
@@ -1014,7 +1014,7 @@ describes.realWin('PeformanceObserver metrics', {amp: true}, (env) => {
       setupFakesForVisibilityStateManipulation();
     });
 
-    it('after performance service registered', async () => {
+    it.only('after performance service registered', async () => {
```
Comment (rcebulko) [inline test/unit/test-performance.js:1017]:
```suggestion
    it('after performance service registered', async () => {
```

**Review by rcebulko — COMMENTED** [review]
Just commenting to flag the `.only` so it isn't forgotten

**Review by erwinmombay — APPROVED** [review]

**Review by rcebulko — APPROVED** [review]

File: src/service/performance-impl.js [hunk 2]
```diff
@@ -353,12 +348,7 @@ export class Performance {
           this.layoutShifts_.push(entry);
         }
       } else if (entry.entryType === 'largest-contentful-paint') {
-        if (entry.loadTime) {
-          this.largestContentfulPaintLoadTime_ = entry.loadTime;
-        }
-        if (entry.renderTime) {
-          this.largestContentfulPaintRenderTime_ = entry.renderTime;
-        }
+        this.largestContentfulPaint_ = entry.startTime;
```
Comment (jridgewell) [inline src/service/performance-impl.js:351]: Do all browsers support `startTime`?

**Review by jridgewell — CHANGES_REQUESTED** [review]
Please update https://github.com/ampproject/amphtml/blob/main/extensions/amp-viewer-integration/TICKEVENTS.md.

File: src/service/performance-impl.js (hunk 2 above)
Comment (samouri) [reply src/service/performance-impl.js:351]: [Yes](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/startTime), this is also how the [web-vitals](https://github.com/GoogleChrome/web-vitals/blob/cfe7536dc8f13e3cbbbf48436328f52399dd6725/src/getLCP.ts#L32-L45) package implements it.

Comment (samouri) [reply src/service/performance-impl.js:351]: @jridgewell: I've updated `TICKEVENTS.md`

**Review by jridgewell — APPROVED** [review]

**Pull request merged**